golem-worker-service-base/src/evaluator/math_op_evaluator.rs

```diff
@@ -1,4 +1,4 @@
-use crate::evaluator::{EvaluationError, EvaluationResult};
+use crate::evaluator::{EvaluationError, ExprEvaluationResult};
```

Comment:
I myself got confused of `EvaluationResult`, mainly on evaluation of `what`?
This is why I renamed this as part of a `fix`